nip47/event_handler.go

```diff
@@ -70,52 +61,50 @@ func (svc *nip47Service) HandleEvent(ctx context.Context, relay nostrmodels.Rela
 			"requestEventNostrId": event.ID,
 			"eventKind":           event.Kind,
 		}).WithError(err).Error("Failed to save nostr event")
-		nip47Response = &models.Response{
-			Error: &models.Error{
-				Code:    constants.ERROR_INTERNAL,
-				Message: fmt.Sprintf("Failed to save nostr event: %s", err.Error()),
-			},
-		}
-		resp, err := svc.CreateResponse(event, nip47Response, nostr.Tags{}, ss)
-		if err != nil {
-			logger.Logger.WithFields(logrus.Fields{
-				"requestEventNostrId": event.ID,
-				"eventKind":           event.Kind,
-			}).WithError(err).Error("Failed to process event")
-		}
-		svc.publishResponseEvent(ctx, relay, &requestEvent, resp, nil)
 		return
 	}
-
 	app := db.App{}
 	err = svc.db.First(&app, &db.App{
-		NostrPubkey: event.PubKey,
+		AppPubkey: event.PubKey,
 	}).Error
 	if err != nil {
 		logger.Logger.WithFields(logrus.Fields{
-			"nostrPubkey": event.PubKey,
+			"appPubkey": event.PubKey,
 		}).WithError(err).Error("Failed to find app for nostr pubkey")
+		return
```

Comment:
If we don't have an app connection for that key anymore we have no way of encrypting the response, do we?
Actually we shouldn't get any requests for that key anyways, since the nostr subscription for that wallet pubKey should no longer exist and we don't get events from the relay for that anymore.

Reply:
Ah true ✅